module/localisation/RobotLocalisation/src/RobotLocalisation.cpp

```diff
@@ -30,7 +30,10 @@
 
 #include "extension/Configuration.hpp"
 
+#include "message/localisation/Field.hpp"
 #include "message/localisation/Robot.hpp"
+#include "message/support/FieldDescription.hpp"
+#include "message/vision/GreenHorizon.hpp"
```

Comment:
Since green horizon, field description and field are all included in the hpp, they don't need to be here

```suggestion
#include "message/localisation/Robot.hpp"
```